server/models/Collective.js

```diff
@@ -680,6 +680,15 @@ export default function (Sequelize, DataTypes) {
             // Prevent collective creation
             throw new Error('Collective creation failed');
           }
+
+          // Set default platformFeePercent
+          if (!instance.platformFeePercent) {
+            if (instance.type === 'COLLECTIVE' || instance.type === 'EVENT') {
```

Comment:
Are we sure that `instance.type === 'COLLECTIVE'` is set at this point and that we're not relying on the default value of the column?

Should we do the opposite and exclude ORGANIZATION and USER` ?

Should EVENT inherit the value of their parent collective?

Reply:
Good question, this seems to be a bit inconsistent.
I think excluding ORG and USER should be safer.